content/en/logs/explorer/search_syntax.md

```diff
@@ -52,9 +50,13 @@ Use the search bar's autocomplete feature to complete your query using:
 
 Clear error states inform you which part of the query contains syntax errors and how to remediate them. For example, if you input the query `service:` with no value, the message "Missing value in key:value pair" is displayed when you hover over the query.
 
-## Escaping of special characters
+## Escape special characters and spaces
 
-The following characters are considered special: `+` `-` `=` `&&` `||` `>` `<` `!` `(` `)` `{` `}` `[` `]` `^` `"` `“` `”` `~` `*` `?` `:` `\`, and `/` require escaping with the `\` character. You can’t search for special characters in a log message. However, you can search for special characters when they’re inside of an attribute. To search for special characters, parse them into an attribute with the [grok parser][1], and then search for logs that contain the attribute.
+The following characters are considered special: `+` `-` `=` `&&` `||` `>` `<` `!` `(` `)` `{` `}` `[` `]` `^` `"` `“` `”` `~` `*` `?` `:` `\`, while spaces and `/` require escaping with the `\` character. 
+
+Instead of searching for special characters in a log message, you can search for special characters when they are inside of an attribute. 
```

Comment:
I would still mention that you can't search for special characters, just so it's clear.

Reply:
```suggestion
You cannot search for special characters in a log message. You can search for special characters when they are inside of an attribute.
```